akd_local_auditor/src/storage/s3/test.rs

```diff
@@ -60,14 +60,40 @@ pub async fn maybe_flush_storage(shared_config: &SdkConfig, bucket_name: &str) -
 
             // From: https://docs.aws.amazon.com/sdk-for-rust/latest/dg/rust_s3_code_examples.html
             let objects = client.list_objects_v2().bucket(bucket_name).send().await?;
-            // let mut delete_objects: Vec<ObjectIdentifier> = vec![];
+
+            let object_versions = client
+                .list_object_versions()
+                .bucket(bucket_name)
+                .max_keys(1000)
+                .send()
+                .await?;
+            let map = object_versions
+                .versions()
+                .unwrap()
+                .iter()
```

Comment:
I think you can call `into_iter()` here since you're not using object versions again.

Reply:
`clippy` actually reverted my `into_iter()`. Since we're calling `.to_string()` anyways, it said we can do it with a borrow.

Reply:
Ah, interesting! Good to know :)